msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -1,40 +1,40 @@
 ---
-title: Receive all channel messages with RSC
+title: Receive all conversation messages with RSC
 author: surbhigupta12
 description: Enable bots to receive all channel messages without being @mentioned using RSC permissions. Read on webApplicationInfo or authorization section in manifest.
 ms.topic: conceptual
 ms.localizationpriority: medium
 ---
 
-# Receive all channel messages with RSC
+# Receive all conversation messages with RSC
 
-The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is extended to bot scenarios.
+The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is being extended to bot scenarios.
 
-Using RSC, you can now request team owners to consent for a bot to receive user messages across standard channels in a team without being @mentioned. This capability is enabled by specifying the `ChannelMessage.Read.Group` permission in the manifest of an RSC enabled Teams app. After configuration, team owners can grant consent during the app installation process.
+Using RSC, you can request team owners to consent for a bot to receive user messages across standard channels and group chats in a team without being @mentioned. The capability is enabled by specifying the `ChannelMessage.Read.Group` or `ChatMessage.Read.Chat` permissions in the manifest of an RSC enabled Teams app. The conversation owners can grant consent during the app installation or upgrade process. Bots can receive all group chat messages in a team when consented to during app installation. For more information about enabling RSC for your app, see [resource-specific consent](../../../graph-api/rsc/resource-specific-consent.md).
 
-For more information about enabling RSC for your app, see [resource-specific consent in Teams](/microsoftteams/platform/graph-api/rsc/resource-specific-consent#update-your-teams-app-manifest).
+## Enable bots to receive all channel or chat messages
 
```

Comment:
Please add notice letting readers know that the ability for bots to receive all messages in _chats_ using `ChatMessage.Read.Chat` is only available in developer preview.

Reply:
Done